pandas/io/parsers/readers.py

```diff
@@ -1100,6 +1110,10 @@ def _clean_options(self, options, engine):
         result["na_values"] = na_values
         result["na_fvalues"] = na_fvalues
         result["skiprows"] = skiprows
+        # Default for squeeze is none since we need to check
+        # if user sets it, we set to False since behavior is
+        # equivlent
```

Comment:
```suggestion
        # Default for squeeze is none since we need to check
        # if user sets it.  We set to False since behavior is
        # equivalent.
```